packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -22,16 +22,18 @@ export const getComboboxStyles = (theme: GrafanaTheme2) => {
     }),
     option: css({
       label: 'grafana-select-option',
+      padding: '8px',
       position: 'absolute',
-      top: 0,
-      left: 0,
-      width: '100%',
+      gridRow: 1,
+      gridColumn: 1,
```

Comment:
```suggestion

```

Reply:
There is no difference in whether to keep or delete this, right?